cmd/aries-js-worker/src/aries.js

```diff
@@ -176,90 +176,255 @@ const Aries = function(opts) {
             return () => {quit = true}
         },
 
+        /**
+         * DIDExchange methods - Refer to [OpenAPI spec](docs/rest/openapi_spec.md#generate-openapi-spec) for
+         * input params and output return json values.
+         */
         didexchange: {
             pkgname: "didexchange",
-            createInvitation: async function (text) {
+
+            /**
+             * Creates a DID Exchange Invitation.
+             *
+             * @param req - json document
+             * @returns {Promise<String>}
+             */
+            createInvitation: async function (req) {
                 return invoke(aw, pending,  this.pkgname, "CreateInvitation", text, "timeout while creating invitation")
             },
+
+            /**
+             * Receives a DID Exchange invitation.
+             *
+             * @param invitation - json document
+             * @returns {Promise<String>}
+             */
             receiveInvitation: async function (invitation) {
                 return invoke(aw, pending,  this.pkgname, "ReceiveInvitation", invitation, "timeout while receiving invitation")
             },
-            acceptInvitation: async function (connectionID, publicDID="") {
+
+            /**
+             * Accepts a DID Exchange invitation.
+             *
+             * @param req - json document
+             * @returns {Promise<String>}
+             */
+            acceptInvitation: async function (req) {
                 return new Promise((resolve, reject) => {
-                    invoke(aw, pending,  this.pkgname, "AcceptInvitation", {id: connectionID, public: publicDID}, "timeout while accepting invitation").then(
+                    invoke(aw, pending,  this.pkgname, "AcceptInvitation", req, "timeout while accepting invitation").then(
                         resp => resolve(resp),
                         err => reject(new Error("failed to accept invitation: " + err.message))
                     )
                 })
             },
-            acceptExchangeRequest: async function (text) {
+
+            /**
+             * Accepts a DID Exchange invitation.
+             *
+             * @param req - json document
+             * @returns {Promise<String>}
+             */
+            acceptExchangeRequest: async function (req) {
                 return invoke(aw, pending,  this.pkgname, "AcceptExchangeRequest", text, "timeout while accepting exchange request")
             },
-            createImplicitInvitation: async function (text) {
+
+            /**
+             * Creates an implicit invitation using inviter DID.
+             *
+             * @param req - json document
+             * @returns {Promise<String>}
+             */
+            createImplicitInvitation: async function (req) {
```

Comment:
rename to `req` not propagated to call to `invoke()`

Reply:
[Done](https://github.com/hyperledger/aries-framework-go/compare/415b0b2413e6b4948632dec5a47ce53406011239..eeb7de2aa3ffaab86cf1542c90f7c65569c8fd2c#diff-a43de5ddb6dcf66800367a9232d43f90R238).